File: benchmark/results/python-invocation-script/run-004/a-mcp/output/inventory/services/stock_service.py
from inventory.aggregates.stock_item import StockItem

"""StockService service."""
class StockService:
    def create(self, input: dict) -> StockItem:
        raise NotImplementedError('not implemented')
    def find_by_id(self, id: str) -> StockItem | None:
        raise NotImplementedError('not implemented')
    def list(self, limit: int) -> list[StockItem]:
        raise NotImplementedError('not implemented')
    def delete(self, id: str) -> None:
        raise NotImplementedError('not implemented')

File: benchmark/results/ts-script-sonnet/run-002/b-baseline/output/src/domain/inventory/services/StockService.ts
import { StockItem } from '../aggregates/StockItem';

/** Service providing business operations for inventory stock. */
export class StockService {
  create(input: Partial<StockItem>): StockItem {
    void input;
    throw new Error('not implemented');
  }

  findById(id: string): StockItem | null {
    void id;
    throw new Error('not implemented');
  }

  list(limit: number): StockItem[] {
    void limit;
    throw new Error('not implemented');
  }

  delete(id: string): void {
    void id;
    throw new Error('not implemented');
  }
}
